packages/prerender-fargate/lib/prerender/server.js

```diff
@@ -36,8 +36,114 @@ server.use({
 });
 
 server.use(prerender.blacklist());
-server.use(prerender.httpHeaders());
-server.use(prerender.removeScriptTags());
-server.use(s3Cache);
+
+if (process.env.ENABLE_REDIRECT_CACHE.toLowerCase() === 'true'){
+    server.use(prerender.removeScriptTags());
+    var he = require('he');
+    var s3 = new (require('aws-sdk')).S3({params:{Bucket: process.env.S3_BUCKET_NAME}});
+    server.use({
+            requestReceived: function(req, res, next) {
+                if(req.method !== 'GET' && req.method !== 'HEAD') {
+                    console.log("skipping requestReceived from S3 Cache... ")
```

Comment:
Minor point but I would remove this console log. It could be useful during local dev but in a prod system it wouldn't be very helpful (e.g. no logging of which request is being skipped).